pkg/awsvpc/sg_analysis.go

```diff
@@ -209,7 +209,11 @@ func (sga *AWSSGAnalyzer) GetSGRule(index int) (
 	}
 	ruleRes.Local = ipblock.GetCidrAll()
 	ruleRes.Index = index
-	return fmt.Sprintf("index: %d, %v", listIndex, ruleStr), ruleRes, isIngress, nil
+	tableName := "IpPermissions's"
+	if !isIngress {
+		tableName = "IpPermissionsEgress's"
+	}
```

Comment:
What are these strings for?

Reply:
To provide context of the index. Once we retrieve the ID properly the index will no longer be needed #767

Reply:
Are these words taken from somewhere?
I don't even understand what they mean.

Reply:
I surely did not invent them... These are the names of the relevant tables; I believed them to be familiar in aws. 
Replaced with inbound/outbound